german/Alkohol/alkohol.md

```diff
@@ -3,4 +3,5 @@ dass Sie angetrunken mit Ihren 3 besten Freunde ins Boot gestiegen sind,
 können sich jedoch seitdem an nichts mehr erinnern. Schließlich hatten Sie acht Tequila.
 Vorab tranken sie bereits 5 Bier.
 
-Was ist passiert?
+Einer der Jungs konnte sich nach ein paar Minuten an bruchstücke errinern. Er wusste das Tommy sich noch auf dem Boot befand, bevor er, am Morgen, nicht da war.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.